### PR TITLE
Creating

### PR DESCRIPTION
**Pull request by DanushMosinyan**

### CODE REVIEW

File: src/Context/CardsColorProvider.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  const changeColor = (index) => {
+    const r = () => (Math.random() * 256) >> 0;
+    const color = `rgb(${r()}, ${r()}, ${r()})`;
+    colors[index] = color;
+    setColors([...colors]);
+  };
+
+  return (
+    <CardsColorContext.Provider
+      value={{
```
Comment (arayik-99) [inline src/Context/CardsColorProvider.jsx]: seperate this to seperate variable

File: src/Context/CardsColorContext.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { useContext, createContext } from "react";
+
+export const CardsColorContext = createContext();
+
+export const useCardsColorContext = () => {
```
Comment (arayik-99) [inline src/Context/CardsColorContext.js]: please shorten this

File: src/Components/CardContainer/index.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import Card from "../Card";
+import "./styles.css";
+
+import { useCardsColorContext } from "../../Context/CardsColorContext";
+
+const CardContainer = () => {
+  const { colors } = useCardsColorContext();
+
+  return (
+    <div className = "cards">
```
Comment (arayik-99) [inline src/Components/CardContainer/index.jsx]: prettify this

**Review by arayik-99 — CHANGES_REQUESTED** [review]
Remove not necessary files

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.body {
```
Comment (Karen-Avdalyan) [inline src/App.css]: change the class name

File: src/App.css [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+  display: flex;
+  flex-direction: row;
```
Comment (Karen-Avdalyan) [inline src/App.css]: delete this row

File: src/App.css [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+  flex-wrap: nowrap;
```
Comment (Karen-Avdalyan) [inline src/App.css]: As I remember you can delete this one also

File: src/Components/CardContainer/styles.css [hunk 7]
```diff
@@ -0,0 +1,7 @@
+.cards {
+  width: 50%;
+  height: 100%;
+  padding :50px;
```
Comment (Karen-Avdalyan) [inline src/Components/CardContainer/styles.css]: prettify this row

File: src/Context/CardsColorContext.js [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+  return useContext(CardsColorContext);
```
Comment (Karen-Avdalyan) [inline src/Context/CardsColorContext.js]: you didn't make it shorter

File: src/Context/CardsColorProvider.jsx [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+    "transparent",
+    "transparent",
+    "transparent",
+    "transparent",
+    "transparent",
+    "transparent",
+  ]);
+
+  const changeColor = (index) => {
+    const r = () => (Math.random() * 256) >> 0;
```
Comment (Karen-Avdalyan) [inline src/Context/CardsColorProvider.jsx]: change the name of variable

**Review by Karen-Avdalyan — CHANGES_REQUESTED** [review]

File: src/Components/CardContainer/index.jsx [hunk 10]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Card from "../Card";
+import "./styles.css";
+
+import { useCardsColorContext } from "../../Context/CardsColorContext";
```
Comment (arayik-99) [inline src/Components/CardContainer/index.jsx:3]: import file after second line

**Review by arayik-99 — CHANGES_REQUESTED** [review]

**Pull request merged**